profiles/default/linux/amd64/17.0/musl/use.force

```diff
@@ -0,0 +1,6 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# Force the flag which corresponds to arch/abi, bug #515130
+amd64
```

Comment:
Likewise.

Reply:
yep.  i got so used to running the whole show myself, i'm not even familiar what's in the other stack layers.